src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -153,7 +151,7 @@ function assembleCssText(tooltipModel: Model<TooltipOption>) {
     cssText.push('box-shadow:' + boxShadow);
     // Animation transition. Do not animate when transitionDuration is 0.
     // If tooltip show arrow, then disable transition
-    transitionDuration
+    !isFirstShow && transitionDuration
         && !(['top', 'left', 'bottom', 'right'].includes(tooltipModel.get('position') as string))
         && tooltipModel.get('trigger') !== 'item'
```

Comment:
Didn't notice it before. But we should use `zrUtil.indexOf` instead of `includes` here. 

Also in the `assembleArrow` function also has `includes` usage. Which should be changed.